docs/DeveloperGuide.md

```diff
@@ -181,7 +181,7 @@ An `Endpoint`,
 
 An `Endpoint` contains the following attributes:
 1. a `Method`, which represents the type of request an API endpoint will send to the server
-2. an `Address`, which represents the address to which the API request is made
+2. an `Address`, which represents the URL address to which the API request is made
```

Comment:
Parameter table in user guide now uses `URL` instead of `Address`, might be good to standardise here as well.

Reply:
heyo I made sure that all address in the Developer's Guide are "url address", cause our class name in the code is still address but I didnt want the readers to be confused between URL and address